tests/resolvers/Mutation/createEvent.spec.ts

```diff
@@ -166,18 +185,18 @@ describe("resolvers -> Mutation -> createEvent", () => {
         organizationId: testOrganization?.id,
         allDay: false,
         description: "newDescription",
-        endDate: new Date().toUTCString(),
+        endDate: new Date("2023-01-29T00:00:00Z"),
         endTime: new Date().toUTCString(),
         isPublic: false,
         isRegisterable: false,
         latitude: 1,
         longitude: 1,
         location: "newLocation",
         recurring: false,
-        startDate: new Date().toUTCString(),
+        startDate: new Date("2023-01-01T00:00:00Z"),
         startTime: new Date().toUTCString(),
         title: "newTitle",
-        recurrance: "DAILY",
+        recurrance: "ONCE",
```

Comment:
Would enums be better here?

Reply:
No it should be string only we used enumerations to define  named constant values. In our case, the recurrance property is expected to have one of the following values: 
"ONCE", "DAILY", "WEEKLY", "MONTHLY","YEARLY".
so we have to write test cases for each